sql_metadb/derived_tables/invoice_voucher_lines_fund_distributions.sql

```diff
@@ -0,0 +1,80 @@
+DROP TABLE IF EXISTS invoice_voucher_lines_fund_distributions;
+-- Create a derived table to extract fund_distributions from invoice_voucher_lines and joining funds related tables
+CREATE TABLE invoice_voucher_lines_fund_distributions AS
+
+WITH funds_distr AS (
+	SELECT
+    	id AS invoice_voucher_line_id,
+        jsonb_extract_path_text(dist.data, 'code') AS fund_distribution_code,
+        jsonb_extract_path_text(dist.data, 'fundId')::uuid AS fund_distribution_id,
+        jsonb_extract_path_text(dist.data, 'invoiceLineId')::uuid AS fund_distribution_invl_id,
+        jsonb_extract_path_text(dist.data, 'expenseClassId')::uuid AS fund_distribution_expense_class_id,
+        jsonb_extract_path_text(dist.data, 'value')::numeric(19,4) AS fund_distribution_value,
+        jsonb_extract_path_text(invvl.jsonb, 'amount')::numeric(19,4) AS invoice_voucher_lines_amount,
+        jsonb_extract_path_text(dist.data, 'distributionType') AS fund_distribution_type,
+        jsonb_extract_path_text(invvl.jsonb, 'externalAccountNumber') AS invoice_voucher_lines_external_account_number,
+        voucherid AS voucher_id
+    FROM
+        folio_invoice.voucher_lines AS invvl
+        CROSS JOIN jsonb_array_elements(jsonb_extract_path(invvl.jsonb, 'fundDistributions')) AS dist(data)
+)
+SELECT
+	invoice_voucher_line_id AS invoice_voucher_line_id,
```

Comment:
Some formatting adjustments are needed such as this tab character.  I will make the changes.